### ISSUE TITLE
Wrong class applied to accordion

### ISSUE THREAD

**Issue by jvalkeejarvi** [nthompson777/ngx-foundation#1]
`panel-group` class is applied to accordion element instead `accordion` class. Styles are incorrect because of wrong class.

I searched the repo and found word panel-group only in following files:
- `bundles/ngx-foundation.es2015.js.map`
- `bundles/ngx-foundation.es2015.js`
- `bundles/ngx-foundation.umd.js.map`
- `bundles/ngx-foundation.umd.min.js.map`
- `bundles/ngx-foundation.umd.min.js`
- `bundles/ngx-foundation.umd.js`
- `accordion/accordion.component.js.map`
- `accordion/accordion.component.metadata.json`

**Comment by jvalkeejarvi** [comment]
Seems like map and metadata files were not updated when related .js files were updated in commit 282444a.

**Comment by nthompson777** [comment]
You are correct with your findings. Thank you for catching this. I will push this update to the repo and NPM later today as a patch.

**Comment by nthompson777** [comment]
Those map files were nuked due to being inaccurate and out of date and were removed for version 1.0.7.

**Comment by nthompson777** [comment]
I'm reopening this bug for 1.0.8 since it is still applicable with this version/patch.

**Comment by jvalkeejarvi** [comment]
Hello again,

is there any news on this? This issue keeps us from using this package at the moment. If there is anything I can  do to help you just tell me.
